backend/danswer/db/standard_answer.py

```diff
@@ -182,30 +187,39 @@ def find_matching_standard_answers(
     query: str,
     db_session: Session,
 ) -> list[StandardAnswer]:
+    """
+    Returns a list of tuples, where each tuple contains a StandardAnswer and a boolean
+    indicating if the query is a regex match.
```

Comment:
Doesn't this function do either a regex match or word matching?

Reply:
Yep, this docstring is from the original PR made by the previous contributor. I'll edit this to match.

How does:

```Returns a list of StandardAnswer definitions matching the query. If `answer_instance.match_regex` is true, the definition is considered "matched" if the query matches the `answer_instance.keyword` using `re.search`. Otherwise, each word in the keyword is matched against the query.```

Also happy to change the behavior in the exact match scenario, I kept the implementation from the original PR because there were no comments on it so I assumed that behavior was acceptable.

Reply:
@yuhongsun96 are you okay with the original word-matching implementation? Happy to change that if desired.